api/contracts.go

```diff
@@ -437,5 +457,5 @@ func NewUniswapv2(ctx context.Context, client *ethclient.Client, root, auth *bin
 		return err
 	}
 
-	return storeBlockResultsForTxs(ctx, client, "uniswapv2_router-swapExactTokensForTokens", tx)
+	return storeBlockResultsForTxs(ctx, client, path, "uniswapv2_router-swapExactTokensForTokens", tx)
```

Comment:
ditto, and also for other
```suggestion
	return storeBlockResultsForTxs(ctx, client, path, "router-swapExactTokensForTokens", tx)
```